Source/Plugins/Core/com.equella.core/src/com/tle/core/freetext/service/FreeTextService.java

```diff
@@ -43,6 +43,18 @@
   <T extends FreetextResult> FreetextSearchResults<T> search(
       Search searchreq, int nStart, int nCount);
 
+  /**
+   * Another search method which allows users to specify if they want to search attachments.
+   *
+   * @param searchReq An search request
+   * @param start The first record of a search result
+   * @param count The number of records of a search result
```

Comment:
```suggestion
   * @param count The maximum number of results requested, or -1 for all.
```
Actually, should this param be called `max` or `length` or `maxCount`/`maxLength`? Ooooh, perhaps not, this is meant to be an overloaded method). In that case these should probably be `nStart` and `nCount` to match the method you're overloading. (Mind you, if you wanted too you could update the other method too so as to improve the names.)